docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.5-slim
+
+#install packages
+#fixes dpkg man page softlink error while installing postgresql-client [source: https://github.com/debuerreotype/debuerreotype/issues/10#issuecomment-438342078]
+RUN mkdir -p /usr/share/man/man1 && mkdir -p /usr/share/man/man7
+RUN apt-get update && apt-get install curl -y
+RUN curl -sL https://deb.nodesource.com/setup_10.x -o nodesource_setup.sh && bash nodesource_setup.sh && rm nodesource_setup.sh
+RUN apt-get install -y --no-install-recommends ruby-dev libarchive-dev libmagickwand-dev libxml2-dev libxslt1-dev build-essential libpq-dev libsasl2-dev imagemagick libffi-dev locales postgresql-client-9.6 nodejs && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN npm install yarn -g
```

Comment:
Docs state `bundle exec rake yarn:install`. If there's a reason to differ, can you add a comment?

Reply:
Different thing - you need to install yarn first (unfortunately it's not packaged in Ubuntu like it is in Fedora) and then `bundle exec rake yarn:install` runs yarn to install the modules listed in `package.json`.